bin/cli.js

```diff
@@ -260,8 +262,8 @@ const onExit = code => {
 };
 
 const config = getConfig();
-console.log('Metatest CLI Stub');
-console.log(`Metatests final config:\n${JSON.stringify(config, null, 2)}\n`);
+if (config.verbosity >= 3)
```

Comment:
I think it'd be better to add the LevelName -> Number mapping and function `isLogAtLeast(verbosity, level)` to make the intent here clear, as using plain numbers here is IMO confusing.